python/lsst/jointcal/jointcal.py

```diff
@@ -161,13 +191,16 @@ def _build_ccdImage(self, dataRef, associations, jointcalControl):
                               dataRef.dataId['visit'], dataRef.dataId['ccd'],
                               dataRef.getButler().get("camera").getName(),
                               jointcalControl)
+        return tanwcs
```

Comment:
There's a blank `return` above (under `if len(goodSrc.sourceCat) == 0`), contrary to this one and the docstring.